content/en/docs/Examples/configuration_files.md

```diff
@@ -0,0 +1,181 @@
+| ``dataset``   | Parent key for holding configuration specific to the dataset files. |
+| ``dimension_map`` | Child key of dataset. A dictionary defining the mappings between input data dimension names and the framework's standardised dimension names.  |
+| ``variable_map`` | Child key of dataset. A dictionary defining the mappings between input data variable names and the framework's standardised variable names. |
+| ``processing_flags`` | A list of strings referring to any preliminary processing methods to be carried out on the data. |
+
+### Example configuration file
+Below is the template of a gridded configuration file:
+```json
+{
+	"type": "gridded",
+	"dimensionality": 3,
+	"chunks": {
+		"dim1":1000,
+		"dim2":1000,
+		"dim3":1000
+	},
+	"grid_ref": {
+		"t-grid": [
+			"glamt",
+			"gphit",
+			"e1t",
+			"e2t",
+			"e3t_0",
+			"deptht_0",
+			"tmask",
+			"bottom_level"
+		]
+	},
+	"dataset": {
+		"dimension_map": {
+			"ex1": "time_counter",
+			"ex2": "deptht",
+			"ex3": "depthu",
+			"ex4": "depthv",
+			"ex5": "y",
+			"ex6": "x",
+			"ex7": "x_grid_T",
+			"ex8": "y_grid_T"
+		},
+		"variable_map": {
+			"ex1": "time_counter",
+			"ex2": "votemper",
+			"ex3": "thetao",
+			"ex4": "temp",
+			"ex5": "toce",
+			"ex6": "so",
+			"ex7": "vosaline",
+			"ex8": "soce",
+			"ex9": "voce",
+			"ex10": "vomecrty",
+			"ex11": "uoce",
+			"ex12": "vozocrtx",
+			"ex13": "sossheig",
+			"ex14": "zos"
+		}
+	},
+	"domain": {
+		"dimension_map": {
+			"ex1": "t",
+			"ex2": "x",
+			"ex3": "y",
+			"ex4": "z"
+		},
+		"variable_map": {
+			"ex1": "time_counter",
+			"ex2": "glamt",
+			"ex3": "glamu",
+			"ex4": "glamv",
+			"ex5": "glamf",
+			"ex6": "gphit",
+			"ex7": "gphiu",
+			"ex8": "gphiv",
+			"ex9": "gphif",
+			"ex10": "e1t",
+			"ex11": "e1u",
+			"ex12": "e1v",
+			"ex13": "e1f",
+			"ex14": "e2t",
+			"ex15": "e2u",
+			"ex16": "e2v",
+			"ex17": "e2f",
+			"ex18": "ff_t",
+			"ex19": "ff_f",
+			"ex20": "e3t_0",
+			"ex21": "e3w_0",
+			"ex22": "e3u_0",
+			"ex23": "e3v_0",
+			"ex24": "e3f_0",
+			"ex25": "tmask",
+			"ex26": "depthf_0",
+			"ex27": "depthu_0",
+			"ex28": "depthv_0",
+			"ex29": "depthw_0",
+			"ex30": "deptht_0",
+			"ex31": "ln_sco",
+			"ex32": "bottom_level"
+		}
+	},
+	"static_variables": {
+		"not_grid_vars": [
+			"jpiglo",
+			"jpjglo",
+			"jpkglo",
+			"jperio",
+			"ln_zco",
+			"ln_zps",
+			"ln_sco",
+			"ln_isfcav"
+		],
+		"coord_vars": [
+			"longitude",
+			"latitude",
+			"time",
+			"depth_0"
+		],
+		"delete_vars": [
+			"nav_lat",
+			"nav_lon",
+			"deptht"
+		]
+	},
+	"processing_flags": [
+		"example_flag1",
+		"example_flag2"
+	]
+}
+```
+Raw __example configuration__ files can be found in the ``config/`` directory within the [COAsT](https://github.com/British-Oceanographic-Data-Centre/COAsT/tree/master/config) github repository.
```

Comment:
As I recall the files in config/ are also templates rather than 'working' or 'raw' files?